### ISSUE TITLE
isEnabled() in NotificationChannelCompat is returning opposite of expected logic

### ISSUE THREAD

**Issue by jsoberg** [lionscribe/notification-channel-compat#2]
For devices running Oreo or above, the `isEnabled()` check in `NotificationChannelCompat` is currently:
```java
    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
        return _oreoNotificationChannel.getImportance() != NotificationManager.IMPORTANCE_NONE;
    }
```
This should instead be checking if our importance is not equal to IMPORTANCE_NONE.

**Comment by jsoberg** [comment]
Somewhat unrelated, but the additional check in NotificationChannelCompat on line 207 (the !isEnabled check) is unnecessary and might cause a crash if isEnabled returns true on a pre API 28 device (since I believe the IMPORTANCE... fields were added in API 28).
```java
... || channel.getImportance() == NotificationManager.IMPORTANCE_NONE
```

**Comment by lionscribe** [comment]
Thank you for report. Bug has been fixed in version 1.0.2 The bug happened due to function originally being called isBlocked, so it was using reverse logic.
